packages/@o3r/eslint-config/package.json

```
@@ -40,7 +40,7 @@
     "@angular-eslint/builder": "~18.4.0",
     "@angular/compiler": "~18.2.0",
     "@eslint-community/eslint-plugin-eslint-comments": "^4.4.0",
-    "@eslint/js": "~9.14.0",
+    "@eslint/js": "<9.15.0",
```

Would it not be safer with a minimal ?
Something like

```suggestion
    "@eslint/js": ">=8.0.0 <9.15.0",
```

---

yes indeed